api/dbadapters/sqldatawarehouse.ts

```diff
@@ -90,25 +95,30 @@ export class SQLDataWarehouseDBAdapter implements IDbAdapter {
   public async evaluate(
     queryOrAction: string | dataform.Table | dataform.Operation | dataform.Assertion
   ) {
-    // TODO: Implement this before using `dbadapter.evaluate` anywhere.
-    if (typeof queryOrAction !== "string") {
-      throw new Error("Evaluate not yet implemented for non strings.");
-    }
-    try {
-      await this.execute(`explain ${queryOrAction}`);
-      return [
-        dataform.QueryEvaluation.create({
-          status: dataform.QueryEvaluation.QueryEvaluationStatus.SUCCESS
-        })
-      ];
-    } catch (e) {
-      return [
-        dataform.QueryEvaluation.create({
+    // TODO: Using `explain` before declaring a variable is not valid in SQL Data Warehouse.
```

Comment:
It seems dry runs with variable declarations don't work. Know of any workarounds?

Reply:
I do not, but I wouldn't worry much about this edge case especially as we have extremely few Azure users